stan/math/rev/core/operator_divide_equal.hpp

```diff
@@ -3,16 +3,18 @@
 
 #include <stan/math/rev/core/var.hpp>
 #include <stan/math/rev/core/operator_division.hpp>
+#include <stan/math/prim/meta.hpp>
 
 namespace stan {
 namespace math {
-
-inline var& var::operator/=(const var& b) {
+template <typename Var, require_var_t<Var>...>
+inline var& var::operator/=(Var&& b) {
```

Comment:
> What is different between require_var_t<Var> and just removing the template parameter and using var instead of Var? I'd think given that it's an overload, we'd also prefer the more specific type to match.

As with the comment above about [universal references](https://isocpp.org/blog/2012/11/universal-references-in-c11-scott-meyers) the `Var&&` and `require_var_t` combo here will match against any `var` type with whatever [deceleration specifiers](https://en.cppreference.com/w/cpp/language/declarations#Specifiers) it has on it (const etc.). So it gives us the most specific type as long as the paremeters decayed type is a `var`.

Does that make sense? ftr I find the universal reference thing in C++the most weird/legalize part of the language